test/posts.test.js

```diff
@@ -172,14 +172,18 @@ describe('test /posts/:id responses', () => {
       .get(`/posts/${addedPost1.id.slice(4)}`)
       .set('Accept', 'text/html');
     expect(res.status).toEqual(400);
-    expect(res.get('content-length')).toContain('34');
   });
 
   test('requests ID with 14 characters Test', async () => {
     const res = await request(app)
       .get(`/posts/${addedPost1.id.concat('1234')}`)
       .set('Accept', 'text/html');
     expect(res.status).toEqual(400);
-    expect(res.get('content-length')).toContain('34');
+  });
+
+  test('requests ID with valid length but not exist Test', async () => {
+    const res = await request(app).get(`/posts/1234567890`).set('Accept', 'text/html');
+    expect(res.status).toEqual(404);
+    expect(res.get('Content-length')).toEqual('46');
```

Comment:
@eunbeek : This looks more like it. One last thing I want to ask from you. Can you squash all of your commit into a single commit? Then resubmit with --force again. I don't want to history to be dirty.

Reply:
#1359 I reopen this PR again for messed commit!